jerry-core/ecma/builtin-objects/ecma-builtin-date-prototype.c

```diff
@@ -551,6 +551,9 @@ ecma_value_t
 ecma_builtin_date_prototype_dispatch_routine (uint16_t builtin_routine_id, /**< built-in wide routine
                                                                             *   identifier */
                                               ecma_value_t this_arg, /**< 'this' argument value */
+                                              const ecma_value_t normalized_arguments_list[3], /**< list of normalized
```

Comment:
I think we can do this without introducing a new argument. Just reuse arguments_list.